package.json

```diff
@@ -24,7 +24,7 @@
   "main": "index.js",
   "scripts": {
     "start": "node example/index.js",
-    "test": "./node_modules/.bin/mocha"
+    "test": "./node_modules/.bin/mocha test/**/*.spec.js test/renderers/**/*.spec.js"
```

Comment:
had to tweak this a bit because I need to run mock tests that generate samples outside of the main test suite.